project_task_stage_change_restriction/models/project_task_stage.py

```
@@ -0,0 +1,35 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+class ProjectTaskStage(models.Model):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)